ktor-client/ktor-client-core/common/src/io/ktor/client/request/forms/FormDataContent.kt

```diff
@@ -31,16 +31,16 @@ public class FormDataContent(
     override fun bytes(): ByteArray = content
 }
 
-
 /**
  * [OutgoingContent] for multipart/form-data formatted request.
  *
  * @param parts: form part data
  */
 public class MultiPartFormDataContent(
-    parts: List<PartData>
+    parts: List<PartData>,
+    val boundary: String = generateBoundary(),
```

Comment:
Could you tell me why do we need `boundary` in the constructor?

Reply:
Hi @e5l I had originally made it override-able but @cy6erGn0m requested an optional constructor argument in the discussion in #1970

Reply:
The other option to only having `contentType` in the constructor and not `boundary` would be a lambda that accepts a string like `contentTypeFn: (String) -> ContentType` but I think that's a little overkill.